README.md

```diff
@@ -30,11 +30,12 @@ A curated list of awesome things and projects built by Algerian developpers.
 - C
   - [ArduinoSpritzCipher](https://github.com/abderraouf-adjal/ArduinoSpritzCipher) - Spritz library for Arduino, CSPRNG, cryptographic hash and MAC functions, symmetric-key data encryption, and general-purpose functions
 - Other languages
-  - [Arabic Stremmer](https://github.com/assem-ch/arabicstemmer) - Arabic Light Stemmer is a snowball-based stemming algorithm for Arabic
+  - [Arabic Stemmer](https://github.com/assem-ch/arabicstemmer) - Arabic Light Stemmer is a snowball-based stemming algorithm for Arabic
 
 ## Other
 - [Deniable Encrypted Keys Database](https://github.com/abderraouf-adjal/dekdb-format-specifications) - Deniable Encrypted Keys Database (DEKDB) format specifications
 - [Tamazight Keyboard Layout](https://github.com/noureddineme/tamazight-layout) - Linux tamazight keyboard layout (bouth Tifinagh and Latin)
+- [DZ Code Academy](https://github.com/Ilyes-Hammadi/dz-code-academy) - Learn to code in Arabic (Algerian dialect)
```

Comment:
This looks like an empty repo pointing to a YouTube channel and missing a license.
Why not link the channel directly in a new _Educational Material_ section, or something like that?

PS: Correct me if I am wrong, but this repo will contain some course material, right? If so, then I don't think it should be included at all.

Last but not least, I do love the work you've done over at that channel! 😉